### PR TITLE
[4.0] com_templates tooltips

### PR DESCRIPTION
**Pull request by brianteeman**
Removes tooltips where they are identical to the item they are a tip for :facepalm

Removes htmlhelper if its just a regular title 

Displays on screen the information about file name restrictions

### CODE REVIEW

**Comment by Quy** [comment]
Remove `use Joomla\CMS\HTML\HTMLHelper;` from template files.

**Comment by brianteeman** [comment]
@quy all done for the files where it was no longer required

**Comment by Quy** [comment]
This can be removed too `class="modalTooltip"`. Sorry did not noticed earlier.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 9ef7d459bcc5d3376b7066343221c5c120509efb<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25231">issues.joomla.org/tracker/joomla-cms/25231</a>.</sub>

File: administrator/components/com_templates/Service/HTML/Templates.php [hunk 1]
```diff
@@ -49,8 +49,7 @@ public function thumb($template, $clientId = 0)
 
 			if (file_exists($preview))
 			{
-				$html = '<button type="button" data-target="#' . $template . '-Modal" class="thumbnail float-left hasTooltip" data-toggle="modal" title="' .
-					HTMLHelper::_('tooltipText', 'COM_TEMPLATES_CLICK_TO_ENLARGE') . '">' . $html . '</button>';
+				$html = '<button type="button" data-target="#' . $template . '-Modal" class="thumbnail float-left" data-toggle="modal" title="'. Text::_('COM_TEMPLATES_CLICK_TO_ENLARGE') . '">' . $html . '</button>';
```
Comment (wilsonge) [inline administrator/components/com_templates/Service/HTML/Templates.php:52]: What's the point of this change? We're removing the tooltip class but leaving the title?

Comment (brianteeman) [reply administrator/components/com_templates/Service/HTML/Templates.php:52]: The class does nothing any more. The title is just displayed as a regular title so there is no need for the extra code as it does nothing

**Comment by wilsonge** [comment]
Thanks!

File: administrator/components/com_templates/Service/HTML/Templates.php (hunk 1 above)
Comment (wilsonge) [reply administrator/components/com_templates/Service/HTML/Templates.php:52]: Ahh missed BS4 had removed the class to trigger a tooltip. Ta

Comment (brianteeman) [reply administrator/components/com_templates/Service/HTML/Templates.php:52]: no problem

**Comment by brianteeman** [comment]
Thanks!

**Pull request merged**